crates/holochain/src/sweettest/sweet_conductor_config_rendezvous.rs

```diff
@@ -77,28 +105,41 @@ impl SweetLocalRendezvous {
             let (turn_addr, turn_srv) = tx5_go_pion_turn::test_turn_server().await.unwrap();
             tracing::info!("RUNNING TURN: {turn_addr:?}");
 
-            let _sig_hnd = sbd_server::SbdServer::new(Arc::new(sbd_server::Config {
-                bind: vec![format!("{addr}:0")],
-                limit_clients: 100,
-                disable_rate_limiting: true,
-                ..Default::default()
-            }))
-            .await
-            .unwrap();
+            let (sig_addr, sig_port, sig_hnd) = spawn_sig(addr, 0).await;
 
-            let sig_addr = *_sig_hnd.bind_addrs().first().unwrap();
-            let sig_addr = format!("ws://{sig_addr}");
-            tracing::info!("RUNNING SIG: {sig_addr:?}");
+            let sig_hnd = Mutex::new(Some(sig_hnd));
 
             Arc::new(Self {
                 bs_addr,
                 bs_shutdown: Some(bs_shutdown),
                 turn_srv: Some(turn_srv),
                 sig_addr,
-                _sig_hnd,
+                sig_hnd,
+                sig_ip: addr,
+                sig_port,
             })
         }
     }
+
+    /// Drop (shutdown) the signal server.
+    #[cfg(feature = "tx5")]
+    pub async fn drop_sig(&self) {
+        self.sig_hnd.lock().unwrap().take();
+
+        // NOTE: on windows (and slow other systems) we need to wait a moment
+        //       to make sure that the old connection is actually closed.
+        tokio::time::sleep(std::time::Duration::from_millis(400)).await;
```

Comment:
Check here that the connection has been closed or put it in a retry loop?

Reply:
done: https://github.com/holochain/holochain/pull/4151/commits/15274f5c53c0ca8350d1f4552954dfd377b9c967